system/jlib/jlog.cpp

```diff
@@ -2645,6 +2647,7 @@ MODULE_EXIT()
     thePassNoneFilter = nullptr;
     thePassLocalFilter = nullptr;
     thePassAllFilter = nullptr;
+    delete logAccessor.queryExisting();
```

Comment:
I suspect this should logAccessor member should also be cleared - but that is a general issue, and I'll create a jira/pr to address it.